config/credentials.yml.enc

```diff
@@ -1 +1 @@
-6gIqb7LxXPP/aag2FhiH+ExJE0Z7k6i/X3qIIcVmXuYGy09l4l/dzV2hXkFr2hZS//FqjbXCueeTP2XHBaOu4FvisNU0UT5/KVnXrj56nUZqOHixAjcp0sZ8UrVi3QuzichKwuBK/Feyh2cnfveZgHQVcOx+dHdAE+aPwwr0GsiKaVyY3R9aZwKPOoW6ULagS6Hb73E+oNLQrNdE5eisrebD9C7BIDw2jIocop4XR+dWdUU41kltCYpv+5huHQAMz4c9Jv1HahDYNKK7lIrkEYg3DocnUYGjAUIwHlI/CxaPJkOmAKob3eg5JHmkY3amBffmZ6lJLDDNiRuGdFopnKgqEiEDvbuvWOlCARU+bv6DaiR0dJK5HrahLrF+ll4WA9aT4XaNxXOXXp55KthTYKJXWwfwduthxG6Q--pjebHAKyAcINVG+o--i/2KUgAIfHLIKMhvXYmWaQ==
\ No newline at end of file
+aKrv1m3B8xtqHq1Rg3f3ffwnLU3u7XLTW7fkmmq3W7/jGacYhEhnTXrp209p5u2IGlrhbxtpYi8vqXbA4kpqJSyCSaBrZuWR+2+r0xpKNhO7A00mC56mKcivNIIlOvably1zgyJqj3PNz/s+3CDz8YC/YzXOvMioFH1nafh3woj1aQ9jI3rYXGO5TU+23ze9ylhbLttXaSiIHxRLdA06kmAL271NbyGtiXFbexwbNCxP9gDxYawliWwA4riUeZGwVqNtfxBUoJ/cekNbnp1sq6RkebNAiUmn0wCbhn2TFvjk1Ym6tXDe6x7wKV3iMThX0WFe2j24g6RkltH060uFxDbZxjec+qPGcVjeyAKzL6DCTO0ak+HqpKe1hh3sxhg4GYAKAOJgusjooS8u6g15RWNcWFmNVD+sOt4/q0kPepEQdPnY1N1O9VrwPHhL018JVlAXfZRzTwrfmlAgbN0kqII9DP6W9dYqFpAopUUjre1/fMS28XKmHZszY+PC2PFajMDMzIkBd92FB6X/ZPqZZAJD+R+ZmTzmiDQBrm6kvcVQu0R1i0Sbd0Sp5WuHWleyOnhV7EQR9fuZ7b1/gXUhWR+kBakNVSMdYJsYxYdIvpD19MqTKc9qQK685iUncE5I+sc5Bn0SyYRHzUedpiSGgWkzCUNV0+aCaTswVcQ6WjsBJWcsrEOK8UrMs8+gILOjquOGwg9h5vNzRagCI4msU/dxf2QVMfk9djFXegj4rTTGZ3/3AF1bTi0yXan8EZ+ydy79mDm1/RHYJnRn0miopAqO51zKPj+IuoN3qXI230ccz6wyA+Iksf1DEYYLugQLhlwLJl29Put/vlkLSH/dqO10xst6MxEeanM2TMBQ9l4tokHtOlbxYiZpxFP/WaMkAwEs1jJfc4xobm4TdyDS1YhTs3WDlRQcnp8L/ngzU14dbIZpQTHBlvEq2xXGh4jKDxvePjB/KUagw0k4AKtacmwzcaerA7TQaZFwBh0TdFtygmUBLNImQ1X8tQ3w2r+BYXkGVKil1UUqTmgrWhKlvRocEH3gB+xfNH0tFa2LSczZE08Zgy4cDIh7qEf9Sfhdk8kt7n7uPcFvDShU06xxnq+hdjon5hc=--EhOO/cIuwO3gyLCe--E9//qEVdA8omSAl/Im5+pA==
```

Comment:
Does this file need to be added to .gitignore and moved to LastPass, or is it safe to keep public?

Reply:
credentials are kept in the repo. they're decrypted with the config/master.key, which is in the gitignore, so they are safe. we keep the master.key in LastPass so we all use the same one. i haven't done that though :-) will do it now